src/iosrv/srv.c

```diff
@@ -237,9 +237,8 @@ dss_ult_sleep(struct dss_sleep_ult *dsu, uint64_t expire_secs)
 	D_ASSERT(thread == dsu->dsu_thread);
 
 	D_ASSERT(d_list_empty(&dsu->dsu_list));
-	daos_gettime_coarse(&now);
-	dsu->dsu_expire_time = now + expire_secs;
-	D_DEBUG(DB_TRACE, "dsu %p expire in "DF_U64" secs\n", dsu, expire_secs);
+	now = daos_get_ntime();
```

Comment:
it is using " clock_gettime(CLOCK_MONOTONIC, ts)".  daos_gettime_coarse() should be faster?

Reply:
yeah, but we do not have get mtime wrap with clock_gettime(), only get_ntime() for the moment. since this is not critical path, so I just use get_ntime() for the moment.

Reply:
hmm, daos_get_ntime is also using clock_gettime(CLOCK_MONOTONIC, ts).